Source/MQTTnet/MqttApplicationMessageExtensions.cs

```diff
@@ -16,6 +17,11 @@ public static string ConvertPayloadToString(this MqttApplicationMessage applicat
                 throw new ArgumentNullException(nameof(applicationMessage));
             }
 
+            if(applicationMessage.PayloadSegment == EmptyBuffer.ArraySegment)
```

Comment:
Nit:
```suggestion
            if (applicationMessage.PayloadSegment == EmptyBuffer.ArraySegment)
```

Reply:
I will make the formatting fix so that I can release a new version today.